src/main/java/org/zalando/automata/aws/AwsEnvironmentPostProcessor.java

```diff
@@ -16,53 +16,57 @@
 import com.amazonaws.util.EC2MetadataUtils;
 
 /**
- *
+ * A simple {@link EnvironmentPostProcessor} for a Spring Boot application that will
+ * recognize when the application is running on AWS EC2 utilizing {@link EC2MetadataUtils} and will
+ * expose various AWS properties on the environment.
  */
 public class AwsEnvironmentPostProcessor implements EnvironmentPostProcessor, Ordered {
 
 	private final Logger log = LoggerFactory.getLogger(AwsEnvironmentPostProcessor.class);
 
 	/**
-	 *
+	 * The base prefix for all exposed properties
 	 */
 	public static final String AWS_PREFIX = "aws";
 
 	/**
-	 *
+	 * The key for the enabled property
 	 */
 	public static final String AWS_ENABLED_KEY = AWS_PREFIX + ".enabled";
 
 	/**
-	 *
+	 * The key for the AWS availability zone property
 	 */
 	public static final String AWS_AZ_KEY = AWS_PREFIX + ".az";
 
 	/**
-	 *
+	 * The key for the AWS region property
 	 */
 	public static final String AWS_REGION_KEY = AWS_PREFIX + ".region";
 
 	/**
-	 *
+	 * The key for the AWS hostname property
 	 */
 	public static final String AWS_HOSTNAME_KEY = AWS_PREFIX + ".localhostname";
 
 	/**
-	 *
+	 * The key for the AWS local IPv4 address property
 	 */
 	public static final String AWS_LOCAL_IPV4_KEY = AWS_PREFIX + ".localipv4";
 
 	/**
-	 *
+	 * The key for the AWS instance ID property
 	 */
 	public static final String AWS_INSTANCE_ID_KEY = AWS_PREFIX + ".instanceid";
 
 	// Before ConfigFileApplicationListener so values there can use these ones
 	private int order = ConfigFileApplicationListener.DEFAULT_ORDER - 1;
 
 	/**
+	 * Set the order for determining when the {@link EnvironmentPostProcessor} should be executed. If
+	 * not set the order will default to {@link ConfigFileApplicationListener} DEFAULT_ORDER - 1.
 	 *
-	 * @param order
+	 * @param order  The int value for the order to set. See {@link Ordered} for more details.
    */
```

Comment:
What would you change here? This one should have two spaces as recommended by http://blog.joda.org/2012/11/javadoc-coding-standards.html

I added the missing double spaces on the return statement below (and changed both param and return to start lower case